Application/Do-Svyazi.User.Dtos/Roles/RoleDto.cs

```diff
@@ -0,0 +1,19 @@
+using Do_Svyazi.User.Domain.Roles;
+
+namespace Do_Svyazi.User.Dtos.Roles;
+
+public class RoleDto
```

Comment:
```suggestion
public record RoleDto
```

и все дто поменяй на рекорд - это улучшит перф :)